runtime/protocol/http-client-engines/test-suite/common/test/aws/smithy/kotlin/runtime/http/test/NonAsciiHeaderValueTest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package aws.smithy.kotlin.runtime.http.test
+
+import aws.smithy.kotlin.runtime.http.HttpStatusCode
+import aws.smithy.kotlin.runtime.http.complete
+import aws.smithy.kotlin.runtime.http.request.HttpRequest
+import aws.smithy.kotlin.runtime.http.test.util.AbstractEngineTest
+import aws.smithy.kotlin.runtime.http.test.util.test
+import aws.smithy.kotlin.runtime.http.test.util.testSetup
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class NonAsciiHeaderValueTest : AbstractEngineTest() {
```

Comment:
**nit/suggestion**: Most of the other tests are under a classification like "uploads" or "downloads". I suggest making this more generic like "HeaderTests"